### PR TITLE
Fix typo in migration guide

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

### CODE REVIEW

File: docs/migrating-v1alpha1-to-v1beta1.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ In Tekton `v1beta`, the following fields have been changed:
...
 | `spec.inputs` | Removed from `Tasks` |
-| `spec.outputs` | Removed from `Tasks` | 
+| `spec.outputs` | Removed from `Tasks` |
 | `spec.inputs.resources` | [`spec.resources.inputs`](#changes-to-pipelineresources) |
 | `spec.outputs.resources` | [`spec.resources.outputs`](#changes-to-pipelineresources) |
 
 ## Changes to input parameters
 
-In Tekton `v1beta`, input parameters have been moved from `spec.inputs.params` to `spec.params`. 
+In Tekton `v1beta`, input parameters have been moved from `spec.inputs.params` to `spec.params`.
```
Comment (sbwsg) [inline docs/migrating-v1alpha1-to-v1beta1.md:31]: Should this be `v1beta1` instead of `v1beta`?

Comment (AlanGreene) [reply docs/migrating-v1alpha1-to-v1beta1.md:31]: It should indeed, good catch!

**Pull request merged**